### ISSUE TITLE
Should imgui_glium_renderer re-export glium?

### ISSUE THREAD

**Issue by nico-abram** [Gekkio/imgui-rs#230]
I think I'm getting a version mismatch with glium using:
```

glium = "0.25"
imgui = "0.0.23"
imgui-glium-renderer = "0.0.23"
imgui-winit-support = "0.0.23"
image = "0.21.2"
```
I think issues like that might be prevented by re-exporting glium in the glium renderer (The same might apply to the winit support crate)

**Comment by Gekkio** [comment]
The version mismatches are indeed very annoying and this idea has been proposed in #121 
However, the proposed solutions have downsides, and in my opinion it would be strange to indirectly pull glium through imgui-glium-renderer. If you build a game, surely your game would depend on glium directly since you'll have a lot of code in your project that uses glium.

What we really need is the opposite: imgui-glium-renderer should use the version of glium your game uses. The same thing also applies to imgui-winit-support ([see my comment here](https://github.com/Gekkio/imgui-rs/issues/214#issuecomment-485246905)). This will be possible once public/private dependencies are handled by Cargo in a better way.

**Comment by Gekkio** [comment]
FWIW, I implemented this in bbf9c57cdc68abb45a301a02ed7c1b9389a16d51 and it will be included in the next major version (0.2.0). I still think it's the wrong solution, but it doesn't cause active harm so it's ok if it helps some people.
